DepositeCalcTests/Tests/SettingsPageTests.cs

```diff
@@ -0,0 +1,151 @@
+﻿using NUnit.Framework;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium;
+using System;
+using DepositeCalcTests.Pages;
+
+namespace DepositeCalcTests.Tests
+{
+    internal class SettingsPageTests
+    {
+        private IWebDriver driver;
+
+        [SetUp]
+        public void Setup()
+        {
+            ChromeOptions options = new ChromeOptions { AcceptInsecureCertificates = true };
+            driver = new ChromeDriver(options);
+            driver.Manage().Timeouts().PageLoad = TimeSpan.FromSeconds(30);
+            driver.Manage().Timeouts().ImplicitWait = TimeSpan.FromSeconds(5);
+            driver.Url = "https://localhost:5001/Settings";
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            driver.Quit();
+        }
+
+        [Test]
+        public void LogoutLinkTest()
+        {
+            // Arrange
+            var settingsPage = new SettingsPage(driver);
+            var loginPage = new LoginPage(driver);
+
+            // Act
+            settingsPage.Logout();
```

Comment:
Look, the 'Logout' method returns LoginPage.
```suggestion

            // Act
            var loginPage = settingsPage.Logout();
```